packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -306,7 +306,7 @@ module.exports = {
                         minimize: true,
                         sourceMap: shouldUseSourceMap,
                         modules: true,
-                        localIdentName: '[path]__[name]___[local]',
+                        localIdentName: '[hash:base64:5]',
```

Comment:
This is still deterministic, it is just smaller and no longer useful for a human to read/look at.

Reply:
> it is just smaller and no longer useful for a human to read/look at

Yes this is exactly the point. Developers should not use those values. And it is nice that they are small.

Reply:
> Developers should not use those values.
I don't agree. Sometimes you have to overwrite some styles but you have no access to this this component (eg component library), so the only way that you have - is to use this localIdentName, eg `[class*="foo__bar__baz"]`